app/src/main/java/org/mifos/mobilebanking/ui/activities/PassCodeActivity.java

```diff
@@ -46,6 +47,7 @@ public int getLogo() {
 
     @Override
     public void startNextActivity() {
+        Toast.makeText(this, "Logged in succesfully", Toast.LENGTH_SHORT).show();
```

Comment:
Don't use hard-coded strings. Write them in strings.xml file and use them. (Do this in the repo that I have commented on the PR)